tools/doc/type-parser.js

```diff
@@ -4,6 +4,7 @@ const jsDocPrefix = 'https://developer.mozilla.org/en-US/docs/Web/JavaScript/';
 
 const jsDataStructuresUrl = `${jsDocPrefix}Data_structures`;
 const jsPrimitives = {
+  bigint: 'BigInt',
```

Comment:
Unfortunately, we cannot add this here, otherwise, the [link would be wrong](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Data_structures#BigInt_type) from this code path:

https://github.com/nodejs/node/blob/cd8f06f64f5fba32cf851de4d59c0e22f45b89c7/tools/doc/type-parser.js#L140-L143

So let us just leave the second one for now.